authors/apps/rate_article/serializer.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import serializers
+from .models import Rate
+
+
+class RateSerializer(serializers.ModelSerializer):
+    """Serializer to rate articles"""
+
+    def validate_rating(self, rating):
+        if rating < 1 or rating > 5:
+            raise serializers.ValidationError("rating should include numbrs 1 to 5")
+        return rating
+
+    class Meta:
+
+        model = Rate
+
+        fields = ('rating', 'user')
```

Comment:
You need to add a line at the bottom of this file.
About the blank lines between this code, there is code that needs compacting. Not only in this file.